### PR TITLE
Contacts tutorial application

### PR DESCRIPTION
**Pull request by arvinth10**

### CODE REVIEW

File: contacts_app/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# contacts_app
```
Comment (yenvanio) [inline contacts_app/README.md]: Update README.md so that it is useful to the reader

File: contacts_app/lib/contactsPage.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      appBar: AppBar(
+        title: (Text('Contacts')),
+      ),
+      body: _contacts != null
+          //Build a list view of all contacts, displaying their avatar and
+          // display name
+          ? ListView.builder(
+              itemCount: _contacts?.length ?? 0,
+              itemBuilder: (BuildContext context, int index) {
+                Contact contact = _contacts?.elementAt(index);
```
Comment (yenvanio) [inline contacts_app/lib/contactsPage.dart:39]: what happens if contacts is null?

**Review by yenvanio — COMMENTED** [review]
looks good just small things

File: contacts_app/lib/contactsPage.dart (hunk 2 above)
Comment (arvinth10) [reply contacts_app/lib/contactsPage.dart:39]: A null contact can't be returned. But if the contacts fields are null (using display name and avatar in this case), null check is applied below

**Pull request merged**